src/Microsoft.IdentityModel.Tokens/RsaCryptoServiceProviderProxy.cs

```diff
@@ -36,6 +36,7 @@ namespace Microsoft.IdentityModel.Tokens
     /// <summary>
     /// The purpose of this class is to ensure that we obtain an RsaCryptoServiceProvider that supports SHA-256 signatures.
     /// If the original RsaCryptoServiceProvider doesn't support SHA-256, we create a new one using the same KeyContainer.
+    /// There is NO support for <see cref="CspParameters"/> and <see cref="CspKeyContainerInfo"/> on non-Windows platforms which makes <see cref="RSACryptoServiceProviderProxy"/> Windows-specific class.
```

Comment:
>// There is NO  [](start = 5, length = 15)

perhaps this should be in a remark section and not in the summary?

Reply:
I thought about this too. Decided to include it in the summary as it may be useful to show this information in an intellisense tooltip (content under remarks section is not shown in the tooltip).

On second thought, a summary section _should_ contain only brief information about a type or member and this one is already quite long. I'll move this sentence under remarks section.

Reply:
no need to capitalize NO here imo.

---
In reply to: [233948740](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1050#discussion_r233948740) [](ancestors = 233948740)